changelog/0065-Populate_repeater_names.yml

```
@@ -0,0 +1,45 @@
+title: Populate repeater names
+key: Populate_repeater_names
+date: 2022-12-13
+optional_per_env: yes
+# (optional) Min version of HQ that MUST be deployed before this change can be rolled out (commit hash)
+min_commcare_version: ec13eaccc55d14b708c4e25349edcfd6936fb4d1
+# (optional) Max version of HQ that can be deployed before this change MUST be rolled out (commit hash)
+max_commcare_version:
+context: |
+  Populate all repeaters' `name` fields with its respective connection settings' name.
+
+  This is optional, but is recommended to do for all environments having more than 200 repeaters. A migration will
+  be added in the future which will automatically apply these changes during a code deploy if they have not yet
+  been applied. If there are many repeaters, it will slow down the migration process.
+
+details: |
+  To prepare for a future change where we allow repeater names to be configurable, we added a `name` field to the
+  `repeaters_repeater` table. In the future change, this field will be popualted with the repeater's corresponding
+  connection settings' name by means of a migration. If there are many repeaters in the environment, this migration
+  might take a long time to complete, so it is recommended to apply the changes before that migration runs in order
+  speed up that migration's execution time.
+
+update_steps: |
+  To see how many repeaters there are in an environment, follow the steps below:
+  
+  Go into the django shell
+  ```shell
+  $ commcare-cloud <env> django-manage shell
+  ```
+
+  Inside the django shell, run
+  ```python
+  $ from corehq.motech.repeaters.models import SQLRepeater
+  $ SQLRepeater.objects.count()
```

nit: the prompt for python shells is not usually `$`, and the prompt character doesn't seem necessary here anyway.
```suggestion
  from corehq.motech.repeaters.models import SQLRepeater
  SQLRepeater.objects.count()
```